types.go

```diff
@@ -52,6 +52,8 @@ type (
 	ConfigAware = launchr.ConfigAware
 	// ManagedFS is a File System managed by launchr.
 	ManagedFS = launchr.ManagedFS
+	// ValueProcessor is an interface for entities that process and transform values
+	ValueProcessor = launchr.ValueProcessor
```

Comment:
If someone wants to rely on ValueProcessor, they can depend on `action.ValueProcessor` which is more readable.
Meaning, we don't need to export it here.

We export here only the types that are defined internal to be exportable and be used in external modules.
The types defined in internal are used internally and prevent circular dependencies.

Reply:
removed